services/app-api/handlers/etl/formTemplateEntitiesUpdate.ts

```diff
@@ -124,6 +124,11 @@ export const getFormTemplateFromS3 = async (
   // try {
   const formTemplateParams: S3Get = {
     Bucket: BUCKET_NAME,
+    /*
```

Comment:
if it no longer works, should we remove it?

Reply:
Yeah, maybe. I think this file's only value at this point is as a template for future ETL scripts. @BearHanded , do you want to keep this around?

Reply:
Doesn't matter too much either way for this PR IMO.